.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,28 @@
+name: Feature Request
+description: "Suggest an idea for this project."
```

Comment:
ditto - maybe ask them to first take a look at existing feature requests and upvote if it's already filed.

Reply:
Resolved as per latest commit.